assets/js/components/Navbar.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import Urls from '../utils/urls';
+import '../../css/normalize.css';
```

Comment:
Is it possible to import these files without using relative import (the ones with `../../`)? It's not a good thing to do, since this makes it harder to find where that file came from.

Reply:
The `normalize` and `base` I've kept there because I only call them here, but `navbar` I've refactored to use `styled-components`. Thank you!